packages/engine/core/README.md

```diff
@@ -143,4 +143,75 @@ manager.activatePlugin(['empty', 'console'])
 
 // Plugin communication
 emptyPlugin.call('console', 'print', 'My message')
-```
\ No newline at end of file
+```
+
+--------------
+
+## Permission
+The Engine comes with a permission system to protect the user from hostile plugins.
+There are two level: 
+- **Global**: at the `PluginManager` level.
+- **Local**: at the `Plugin` level.
+
+### Global Permission
+Communication between plugins goes through the `PluginManager`'s permission system : 
+
+```typescript
+canActivatePlugin(from: Profile, to: Profile): Promise<boolean>
+```
+Used when a plugin attempts to activate another one. By default when a plugin "A" calls plugin "B", if "B" is not deactivated, "A" will attempt to active it before perform the call. 
```

Comment:
Used when a plugin attempts to activate another one.  By default when plugin "A" calls plugin "B", if "B" is not deactivated, "A" will attempt to active it before performing the call.